src/Yarhl.Media/Text/Encodings/EscapeOutRangeEncoding.cs

```diff
@@ -284,8 +284,10 @@ static bool MatchSequence(IList<byte> buffer, int index, params byte[] sequence)
                 return false;
 
             for (int i = 0; i < sequence.Length; i++)
+            {
```

Comment:
Braches should go in the same line for loops and conditions [rule](https://github.com/SceneGate/Yarhl/blob/master/CONTRIBUTING.md#brace-position)